wordpress-mobile/WordPress-Android
# Remove recommended sites from manage topics and sites

Since we now display recommended sites in the reader discover feed in https://github.com/wordpress-mobile/WordPress-Android/pull/13162, this PR removes "Sites you may like" tab (and associated logic) from the "Manage Topics and Sites" screen to have consistent behaviour with iOS app.

### To test

- Go to Reader
- Click Settings icon on the app bar
- Notice that "Sites you may like" section is no longer visible

Also notice that `tbl_recommended_blogs` does not exist in the `wpreader.db `using the AS -> database inspector.

Before | After
-------|------
![before](https://user-images.githubusercontent.com/1405144/97400026-60435d80-1914-11eb-9384-50bd2fe59d1f.png) | ![after](https://user-images.githubusercontent.com/1405144/97400491-38082e80-1915-11eb-877b-9d6e951d2d02.png)

PR submission checklist:
